src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/actionpostponementrequest/ActionPostponementRequestAboutToSubmitHandlerTest.java

```diff
@@ -188,6 +188,7 @@ public void givenARefusedPostponement_thenClearReviewStateAndReferralReasonAndFl
 
     @Test
     public void givenAGrantedPostponement_thenClearReviewStateAndReferralReasonAndFlagAndAddNoteAndDwpStateAndDecisionDocAdded() {
+        sscsCaseData.getPostponementRequest().setListingOption("readyToList");
```

Comment:
Please save "readyToList" as a constant, or use State.READY_TO_LIST.getType(), and replace all usages in the test class